templates/openj9/sso74-openj9-https.json

```diff
@@ -0,0 +1,579 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass" : "icon-sso",
+            "tags" : "sso,keycloak,jboss,hidden",
+            "version": "v7.4.0.GA",
```

Comment:
@mhajas Please undo the `version` & `rhsso` `s/7.4.0.GA/v7.4.0.GA/g` replacements. @hmlnarik is right, this was never used in RH-SSO 7.3 yet. I confused it yesterday with the tag name format (too many issues to handle yesterday at once, so overlooked this).

Sorry for the confusion (additional work). Thought I closed all the wrong comments yesterday already, but it was obviously too late